src/main/java/racingcar/domain/Judgement.java

```diff
@@ -0,0 +1,71 @@
+package racingcar.domain;
+
+import static racingcar.common.info.RacingGameInfo.*;
+
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.IOException;
+import java.io.OutputStreamWriter;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Stream;
+import racingcar.common.info.RacingGameInfo;
+
+public class Judgement {
+
+    private static final String DELIMITER = " : ";
+    private final BufferedWriter log = new BufferedWriter(new OutputStreamWriter(System.out));
+
+    public void judge(final List<Car> cars) {
+        int max = maxDriveLength(cars);
+
+        try{
+            log.write(WINNER_IS);
+            log.flush();
+        }catch(IOException exception){
+            System.out.println(exception.getMessage());
+        }
+
+        for(int idx=0; idx<cars.size(); idx++){
```

Comment:
인덱스가 필요하지 않은 경우에는 다음과 같이도 사용할 수 있습니다 :) 
```suggestion
        for (Car car : cars) {
```

Reply:
반영하였습니다.!! ㅜㅜ 꼼꼼하지 못하네요.. 감사합니다. :)